moddingway/commands/note_commands.py

```diff
@@ -0,0 +1,104 @@
+import discord
+from discord.ext.commands import Bot
+from util import is_user_moderator
+from .helper import create_logging_embed, create_response_context
+from services import note_service
+from database import notes_database
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+def create_note_commands(bot: Bot) -> None:
+    @bot.tree.command()
+    @discord.app_commands.check(is_user_moderator)
+    @discord.app_commands.describe(user="User to add note to")
+    @discord.app_commands.describe(note="Note for user, no more than 300 characters")
+    async def add_note(
+        interaction: discord.Interaction,
+        user: discord.Member,
+        note: str,
+    ):
+        """Add a note to the user"""
+        async with create_response_context(interaction) as response_message:
+            async with create_logging_embed(
+                interaction, user=user, note=note
+            ) as logging_embed:
+                await note_service.add_note(logging_embed, user, note, interaction.user)
+
+                response_message.set_string(
+                    f"Successfully added note to  {user.mention}"
+                )
+
+    @bot.tree.command()
+    @discord.app_commands.check(is_user_moderator)
+    @discord.app_commands.describe(user="User whose notes you are viewing")
+    async def view_notes(interaction: discord.Interaction, user: discord.Member):
+        """View the notes of the user"""
+        async with create_response_context(interaction) as response_message:
+            note_details = await note_service.get_user_notes(user)
+            response_message.set_string(note_details)
+
+    class MyView(discord.ui.View):
```

Comment:
Can you rename this to something along the lines of "note delete view"? Also, I think it should be moved out of the `create_note_commands` method unless that breaks something